data/tokens.json

```diff
@@ -1499,30 +1499,6 @@
       }
     }
   },
-  "Accordion": {
-    "gtmds_helper_accordion_border_open": {
```

Comment:
@tsnikolov  We use "_" to separate categories and "-" to define a variant.
I don't see why we should have two tokens for the border since it is the same.
So this should be something like "gtmds_accordion_border", which is enough